node_modules/oae-principals/lib/internal/dao.js

```diff
@@ -82,8 +83,14 @@ var createUser = module.exports.createUser = function(user, callback) {
  */
 var createGroup = module.exports.createGroup = function(groupId, tenantAlias, displayName, description, visibility, joinable, createdBy, callback) {
     var lastModified = Date.now();
+    var created = new Date(lastModified);
+
+    lastModified = lastModified.toString();
+    created = created.toUTCString();
```

Comment:
I think that's just formatting, not time zone conversion. But I had to change that because of the new driver since those two fields are different:

```
CREATE TABLE "Principals" (
...
  created timestamp,
....
  "lastModified" text,
...
  PRIMARY KEY ("principalId")
```

...otherwise, the DB wouldn't accept the created value as it wouldn't match the expected type.